taipy/core/_manager/_manager.py

```diff
@@ -125,11 +126,16 @@ def _get(cls, entity: Union[str, EntityType], default=None) -> EntityType:
             return default
 
     @classmethod
-    def _exists(cls, entity_id: str) -> bool:
+    def _exists(cls, entity_id: str) -> ReasonCollection:
         """
         Returns True if the entity id exists.
```

Comment:
Do we also need to update the docstring for the return type?

The `is_ready_to_run()` method did get an update, but not the others

Reply:
yep I believe we should, let me add that in